### ISSUE TITLE
feature request: pagination (to handle large datasets)

### ISSUE THREAD

**Issue by vrtareg** [metagriffin/firecsv#1]
Hi

I just installed your PlugIn and it crashing my Firefox.

I have 36.0 Beta installed.

Please see error attached.

![firecsv - hung](https://cloud.githubusercontent.com/assets/2562730/6149029/b5a3ab24-b1fd-11e4-83a8-cd83a8b6fe95.PNG)

Regards,
Areg

**Comment by metagriffin** [comment]
thanks for reporting this. can you provide the CSV file that you were looking at when this occurred?

**Comment by vrtareg** [comment]
Hi

Looks like I understood the problem - the csv file is quite big - 25Mb.
This causes PlugIn and Firefox to crash ....

Here is the file - https://drive.google.com/file/d/0BwrDrtwvCRDHY3F1NGlYNUNRRXc/view?usp=sharing

Regards,
Areg

**Comment by metagriffin** [comment]
yes, it is indeed a file-size issue... i have thought about adding a pagination feature, which "should" address this problem. unfortunately, that significantly complicates the machinery (e.g. sorting, graphing, etc.). i'll re-title this issue, and add it to the "feature wishlist".

alternatively, if you are inspired, i'd be happy to accept any PRs!
